Couchbase.AspNet/app.config

```diff
@@ -1,10 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <configSections>
+    <section name="log4net" type="log4net.Config.Log4NetConfigurationSectionHandler, log4net" />
+    <section name="logging" type="Common.Logging.ConfigurationSectionHandler, Common.Logging" />
+  </configSections>
+
+  <traceConfiguration logFolder="C:\logs" logFileNameBase="tracing-Cvent.Web.Subscribers" minimumLoggingLevel="DEBUG" />
```

Comment:
We shouldn't have consumer-specific information here. This package is used across many different web applications.